test/utils/wabt_engine.cpp

```diff
@@ -58,13 +58,19 @@ void WabtEngine::instantiate() {}
 
 bytes_view WabtEngine::get_memory() const
 {
+    if (m_env.GetMemoryCount() == 0)
+        return {};
+
     auto& env = const_cast<wabt::interp::Environment&>(m_env);
     const auto& memory = env.GetMemory(0);
     return {reinterpret_cast<uint8_t*>(memory->data.data()), memory->data.size()};
 }
 
 void WabtEngine::set_memory(bytes_view memory)
 {
+    if (memory.empty())
+        return;
+
```

Comment:
Shouldn't here be an assert for `GetMemoryCount() != 0` ?

Reply:
There is the assert like this inside `GetMemory()`.